Makefile

```diff
@@ -17,14 +17,14 @@ KUBECONFIG ?= $(ROOT_DIR)/admin.conf
 
 DOCKER_REPO_NAME ?= mesosphere
 
-SPARK_IMAGE_NAME ?= spark-2.4.4-bin-hadoop2.7-k8s
+SPARK_IMAGE_NAME ?= spark-k8s
 SPARK_IMAGE_DOCKERFILE_PATH ?= $(ROOT_DIR)/images/spark
-SPARK_IMAGE_TAG ?= $(call get_sha1sum,$(SPARK_IMAGE_DOCKERFILE_PATH)/Dockerfile)
+SPARK_IMAGE_TAG ?= 2.4.4-hadoop2.7
 SPARK_IMAGE_FULL_NAME ?= $(DOCKER_REPO_NAME)/$(SPARK_IMAGE_NAME):$(SPARK_IMAGE_TAG)
 
 OPERATOR_IMAGE_NAME ?= kudo-spark-operator
 OPERATOR_DOCKERFILE_PATH ?= $(ROOT_DIR)/images/operator
-OPERATOR_VERSION ?= $(call get_sha1sum,$(OPERATOR_DOCKERFILE_PATH)/Dockerfile)
+OPERATOR_VERSION ?= $(shell cat VERSION)
```

Comment:
as discussed during the meeting if we want to fix the issue related to the fact that single Dockerfile checksum doesn't represent the full contents of the image: other files which copied to the image and operator git commit SHA. So to make this work properly we can stick to the following approach:
- use checksum of all files in the folder (for Operator image it should include parent Spark image checksum)
-  concatenate that computed checksum with Git commit SHA of Spark Operator submodule

Here's a possible implementation of the folder SHA:
```
# to avoid inconsistencies with files ordering in Linux/Unix systems the final checksum is
# calculated based on sorted checksums of independent files stored in a temporary file
files_checksum() {
    files=("$@")
    echo > tmp.checksum
    if [[ -n "$@" ]]; then
        for file in "${files[@]}"; do
            sha1sum ${file} | cut -d' ' -f1 >> tmp.checksum
        done
    fi
    sort tmp.checksum | sha1sum | cut -d' ' -f1
}
```

which can be invoked as follows: `$(files_checksum $(find "${OPERATOR_DOCKERFILE_PATH}/shared" -type f))`

Reply:
Anton (@akirillov), thanks for spent additional time to discuss this (minor) stuff during the meeting! PTAL^